spec/system/managing_topics_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'managing topics' do
+  before do
+    FactoryBot.create(:topic, name: 'What is your favourite colour')
+  end
+
+  scenario 'when viewing' do
+    visit('/topics')
+    expect(page).to have_content('Topics')
+    expect(page).to have_content('What is your favourite colour')
+  end
+
+  scenario 'can navigate to new topic page' do
+    visit('/topics')
+    expect(page).to have_link('Add your own topics here',href: new_topic_path)
+  end
+
+  scenario 'when creating a topic' do
+    visit '/topics/new'
+    fill_in 'Name', with: 'tell us a story'
+    click_button 'Create'
+    expect(page).to have_content 'tell us a story'
+  end
+
+  scenario 'when forgetting to complete add my topic' do
+    visit '/topics/new'
+    click_button 'Create'
+
+    message = page.find("#topic_name").native.attribute("validationMessage")
```

Comment:
Test for the presence of the model validation message (i.e. "can't be blank").
(As discussed in another comment, don't rely on browser validation)